source/guide_directus.rst

```diff
@@ -0,0 +1,154 @@
+.. highlight:: console
+
+.. author:: j3n57h0m45 <https://github.com/j3n57h0m45>
+
+
+.. tag:: self-hosting
+.. tag:: database
+
+.. sidebar:: About
+
+  .. image:: _static/images/directus.png
+        :align: center
+
+########
+Directus
+########
+
+.. tag_list::
+
+Directus_ is a real-time API and App dashboard for managing SQL database content.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`the shell <basics-shell>`
+  * :manual:`Node.js <lang-nodejs>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+All relevant legal information can be found here in the GitHub_ repository of the project:
+
+  * https://github.com/directus/directus/blob/main/license
+
+Prerequisites
+=============
+
+n/a
+
+Installation
+============
+
+Create directory ``directus``.
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/directus/logs && 
+ [isabell@stardust ~]$ cd ~/directus
+ [isabell@stardust ~]$
+
+.. note::
+
+  As CentOS 7 (uberspace) only provides ``glibc 2.17``, the ``argon2`` module on needs to be manually recompiled. This will avoid the ``Error: /lib64/libc.so.6: version GLIBC_2.25``.
+    
+  .. code-block:: bash
+    
+    [isabell@stardust ~]$ cd ~/directus
+    [isabell@stardust ~]$ npm install @mapbox/node-pre-gyp
+    [isabell@stardust ~]$ npm install argon2 --ignore-scripts
+    [isabell@stardust ~]$ npx node-pre-gyp rebuild -C ./node_modules/argon2
```

Comment:
so i guess ``npx``  is not part of a verbatim uberspace?

Reply:
To get around the ``npx`` issue and avoid pollution the global modules I rewrote the procedure. However if ``npx`` is available, this could simplify the procedure.

Reply:
added note on perquisites to have it available